pkg/kademlia/testrouting/viz.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testrouting
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io"
+)
+
+// Graph writes a DOT format visual graph description of the routing table to w
+func (t *Table) Graph(w io.Writer) error {
+	_, err := w.Write([]byte("digraph{node [shape=box];"))
+	if err != nil {
+		return err
+	}
+	err = t.graph(w, t.makeTree())
+	if err != nil {
+		return err
+	}
+	_, err = w.Write([]byte("}\n"))
+	return err
+}
+
+func (t *Table) graph(w io.Writer, b *bucket) error {
+	if b.split {
+		_, err := fmt.Fprintf(w, "b%s [label=\"%s\"];", b.prefix, b.prefix)
```

Comment:
Instead of `\"%s\"` use `%q`.

Reply:
hmm, i wonder what the escape sequence language is in dot. it's probably not the same as go's but sure, this is strictly better

Reply:
in retrospect i'm embarrassed by my increased likelihoods of sql injection style problems here, but in my defense, b.prefix is always alphanumeric

Reply:
I'm aware. The problem is that people copy-paste code. i.e. it's better to use `"%q"` or `strconv.Quote`... a comment is also fine, but it should be clear either that "it's properly quoting" or the value is sanitized, e.g. `sanitized(b.prefix)` or something.

It's more about good habits rather than this particular issue.

Reply:
As for dot syntax, it's such that Go quotation should work:

```
_escaped_char : '\\' ( _unicode_char | '"' | '\\' ) ;
_char         : _unicode_char | _escaped_char ;
_string_lit   : '"' { _char } '"' ;
```